airbyte-webapp/src/packages/cloud/lib/domain/users/UserService.ts

```diff
@@ -34,8 +34,11 @@ class UserService extends AirbyteRequestService {
     return this.fetch<User>(`web_backend/users/create`, user);
   }
 
-  public async remove(userId: string): Promise<void> {
-    return this.fetch(`${this.url}/delete`, { userId });
+  public async remove(workspaceId: string, email: string): Promise<void> {
+    return this.fetch(`web_backend/cloud_workspaces/revoke_user`, {
```

Comment:
It should use then this.rootUrl before web_backend. Otherwise it won’t be able to specify another host form config.

Reply:
I was wrong about my comment here.